src/main/java/com/minecolonies/coremod/commands/AbstractCommandParser.java

```diff
@@ -0,0 +1,675 @@
+package com.minecolonies.coremod.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'jline.internal.Log'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)